### PR TITLE
Make web3j-corda windows compatible

### PR DESCRIPTION
**Pull request by Puneetha17**
To be verified in a windows system

### CODE REVIEW

File: codegen/src/main/kotlin/org/web3j/corda/codegen/CorDappGenerator.kt [hunk 1]
```diff
@@ -151,6 +153,9 @@ class CorDappGenerator(
         } ?: throw IllegalStateException("Template not found: $TEMPLATE_DIR/$file")
     }
 
+    private val stubPath = "%s${File.separator}src${File.separator}%s${File.separator}kotlin${File.separator}" +
```
[the remark this answers is not in the document]
Comment (xaviarias) [reply codegen/src/main/kotlin/org/web3j/corda/codegen/CorDappGenerator.kt]: Or even: `listOf("a", "b", "c").joinToString(File.separator)`

File: codegen/src/main/resources/cordapp/cordapp_gradle.mustache [hunk 2]
```diff
@@ -4,7 +4,7 @@ buildscript {
         corda_release_group = 'net.corda'
         corda_release_version = '4.1'
         corda_gradle_plugins_version = '4.0.45'
-        kotlin_version = '1.2.71'
+        kotlin_version = '1.3.50'
```
Comment (xaviarias) [inline codegen/src/main/resources/cordapp/cordapp_gradle.mustache]: Not sure if we can upgrade the Kotlin version, Corda only supports 1.2.71 so the generated CorDapp should keep this version no?

File: codegen/src/main/resources/cordapp/cordapp_gradle.mustache [hunk 3]
```diff
@@ -43,6 +56,7 @@ allprojects {
 
     repositories {
         mavenCentral()
+        mavenLocal()
```
Comment (xaviarias) [inline codegen/src/main/resources/cordapp/cordapp_gradle.mustache]: Is that required for some reason?

Comment (Puneetha17) [reply codegen/src/main/resources/cordapp/cordapp_gradle.mustache]: Ah no, that was for my local testing, sorry, will remove it.

**Pull request merged**